@dilirity Changes look good, but phan is complaining:
```
projects/plugins/boost/app/lib/minify/class-concatenate-js.php:370
Error: Plugin PhanPluginSimplifyExpressionBool (false === $processor->next_tag()) can probably be simplified to !($processor->next_tag())

FAQ on Phan issues: pdWQjU-Jb-p2
Error: Process completed with exit code 1.
```

--- a/projects/plugins/boost/app/lib/minify/class-concatenate-js.php
+++ b/projects/plugins/boost/app/lib/minify/class-concatenate-js.php
@@ -367,7 +367,7 @@ class Concatenate_JS extends WP_Scripts {
 		$processor  = new \WP_HTML_Tag_Processor( $script_tag );
 
 		// If for some reason the script tag isn't valid, bail.
-		if ( false === $processor->next_tag() ) {
+		if ( ! $processor->next_tag() ) {
 			return false;
 		}
 

